docs/guide/install.md

```diff
@@ -37,3 +37,61 @@ $ npx slidev
 > ```bash
 > echo 'shamefully-flatten=true' >> .npmrc
 > ```
+
+## Install on Docker
+
+If you need a rapid way to run a presentation, you can use the prebuilt (or even build your own) [docker](https://hub.docker.com/r/stig124/slidev) image (maintained by [stig124](https://github.com/Stig124))
+
+##### *(this is not meant as a developper environnement, as it uses only npm-fetched binaries)*
+
+Pull the image :
+
+```bash
+$ docker pull stig124/slidev:latest
+```
+
+> [podman](https://podman.io/) users can pull the image using the complete URL
+>
+> ```bash
+> $ podman pull docker.io/stig124/slidev:latest
+> ```
+
+
+Go into a directory with a `slides.md` file and run it :
+
+```bash
+$ docker run --rm -v .:/root/slides -p 3000:3030 stig124/slidev:latest
+```
+
+> You can change the port that will face outside by changing the number left of the port declaration => `3000:3030`
+
+Open a browser on `localhost:3000` and you're ready to go, you can leave at any time using `Ctrl` + `C`
+
+> Please note that if you have a `node_modules` folder within the folder with the `slides.md` file, It will be **deleted** *(You can add `-e KEEP=1` next to `--rm` in the command to revert this behaviour)* as it may blocks the starting of the container
+
+### Build the docker image
+
+If you want to build the image by yourself, you can :
```

Comment:
```suggestion
If you want to build the image by yourself from source, you can:
```

Reply:
This is not a build from source, just the Dockerfile is

Reply:
Ah, then it's might not need to clone the main repo, otherwise this could introduce some confusing, as the npm installs might behavior differently than local source. Or we could move the script to another repo in the org that just contains those files.

Reply:
We can

Reply:
Alright, what the common name of it? `slidevjs/dockfile`? I can create the repo and invite you

Reply:
I would prefer `container`, but `docker` or `image` would be a good name

Reply:
https://github.com/antfu/container Here you go

Reply:
Here you go https://github.com/slidevjs/container